api/envoy/extensions/filters/http/cache/v3/cache_policy.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
```

Comment:
This is not an HTTP filter config, rather the config for a cache policy example, it should belong at top-level, under something like `api/envoy/extensions/http/cache_policy`

Reply:
This belongs in `api/envoy/extensions/cache_policy/cache_policy/v3/config.proto` actually, to parallel the existing cache storage extension. I assume this is some simple nop cache policy? You should add more descriptive config proto comments.